internal/querycoord/errors.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2019-2020 Zilliz. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License
+// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+// or implied. See the License for the specific language governing permissions and limitations under the License.
+
+package querycoord
+
+import (
+	"fmt"
+)
+
+func errQueryNodeIsNotOnService(id UniqueID) error {
+	return fmt.Errorf("query node %d is not on service", id)
+}
+
+func msgQueryCoordIsUnhealthy(coordID UniqueID) string {
```

Comment:
msgQueryCoordIsUnhealthy duplicated with function errQueryCoordIsUnhealthy.

Reply:
My mistake here. I will fix this soon.